src/app/app-routing.module.ts

```diff
@@ -1,7 +1,15 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 
-const routes: Routes = [];
+import { NotFoundComponent } from './youtube/pages/not-found/not-found.component';
+import { Auth2Guard } from './auth/guards/auth2.guard';
+
+const routes: Routes = [
+  { path: 'home', loadChildren: () => import('./youtube/youtube.module').then(m => m.YoutubeModule), canLoad: [Auth2Guard] },
+  { path: '', redirectTo: 'home', pathMatch: 'full', canLoad: [Auth2Guard] },
+  { path: 'login', loadChildren: () => import('./auth/auth.module').then(m => m.AuthModule) },
```

Comment:
you could store all base routes to enum like 
export enum AppRoute = {
Login = 'login',
Home ='home,
Empty = ''
}

and then use it in all parts of the app, it will help you to change any existing rout without no pain